### DISCUSSION TITLE
Make a Wilder Wilder Survival server

### DISCUSSION THREAD

**Discussion by mapleexists** [FrozenBlock/WilderWild#145] [Ideas]
that would be pretty fun i think
idk who would host it though
maybe we can convince enough people from the community to donate to us so we can afford a server idk

**Comment by EthanRStokes** [comment]
yes

**Comment by AViewFromTheTop** [comment]
Agreed
Cue me trying to get as far away from everyone as possible and making a secret base, then slowly depleting others of their resources so they have no way to combat me

**Reply by AViewFromTheTop** [reply]
I’ve done this before and got almost full Netherite within a few days, and got banned from the realm because other people felt “threatened”

**Reply by mapleexists** [reply]
LOL

**Reply by EthanRStokes** [reply]
i am going to follow you lunade

**Reply by AViewFromTheTop** [reply]
> i am going to follow you lunade

Why
And where

**Reply by Osmiooo** [reply]
> Why And where

IRL

**Reply by EthanRStokes** [reply]
> > i am going to follow you lunade
> 
> Why And where

so you cant run away and in minecraft

**Reply by AViewFromTheTop** [reply]
I’m good at Sword PvP so good frickin luck

**Reply by AViewFromTheTop** [reply]
so

**Reply by mapleexists** [reply]
none of us would be able to host it lol
